acceptance/suites/tests/555-puppetserver-service/reboot.rb

```diff
@@ -0,0 +1,55 @@
+if (options[:type] == 'pe') then
+  pssn='pe-puppetserver'
+else
+  pssn='puppetserver'
+end
+
+test_name "Validates the ability of the #{pssn} service to reliably start after a reboot."
+
+max=3
+sleeptime=45
+retryCounter=0
```

Comment:
As we gain more confidence that we aren't seeing problems with our service running after reboot, we can turn max (max reboots) down from 3 to 1.  I started at 25 and have been turning it down as I run it more times and don't see problems.